tool/docker/devel/native/ubuntu/cuda/py3/Dockerfile

```diff
@@ -45,16 +46,21 @@ RUN apt-get update \
         numpy \
         setuptools \
         protobuf \
-        future \
-    # install swig > 3.0.10 for ubuntu < 16.04
-    # RUN wget http://prdownloads.sourceforge.net/swig/swig-3.0.10.tar.gz && \
-    #     tar zxf swig-3.0.10.tar.gz && cd swig-3.0.10 && \
-    #     ./configure && make && make install
-    # set environment
+        future
+# install swig > 3.0.10
+RUN wget http://prdownloads.sourceforge.net/swig/swig-3.0.10.tar.gz -P /tmp/ \
+    && tar zxf /tmp/swig-3.0.10.tar.gz -C /tmp/ \
+    && cd /tmp/swig-3.0.10 && ./configure && make && make install
     # ENV CMAKE_INCLUDE_PATH /usr/local/cuda/include:${CMAKE_INCLUDE_PATH}
     # ENV CMAKE_LIBRARY_PATH /usr/local/cuda/lib64:${CMAKE_LIBRARY_PATH}
     # config ssh service
-    && mkdir /var/run/sshd \
+# install mkldnn
+RUN git clone https://github.com/intel/mkl-dnn.git /tmp/mkl-dnn \
```

Comment:
it would be better to fix the version, e.g., from https://github.com/intel/mkl-dnn/archive/v1.0-pc.zip
otherwise, when the github repo is updated with API changes; the compilation of singa may fail.

Reply:
thank you, this is right, I have fixed it.